ibu-imager/postpivot/postpivot.go

```diff
@@ -246,3 +252,22 @@ func (p *PostPivot) applyManifests() error {
 	}
 	return nil
 }
+
+func (p *PostPivot) recoverLvmDevices() error {
+	lvmConfigPath := path.Join(p.workingDir, common.LvmConfigDir)
```

Comment:
can it be run couple of times? If not please use utils.runonce as service can be restarted